src/borg/testsuite/archiver.py

```diff
@@ -79,6 +79,7 @@ def exec_cmd(*args, archiver=None, fork=False, exe=None, **kw):
                 archiver = Archiver()
             archiver.prerun_checks = lambda *args: None
             archiver.exit_code = EXIT_SUCCESS
+            helpers.return_code = EXIT_SUCCESS
```

Comment:
hmm, guess there is no special reason why return_code global should live in helpers module while exit_code lives in archiver module.

so, how about: have both in archiver and make naming clearer (or add comments what they are for).